Sources/MongoSwift/MongoCollection+BulkWrite.swift

```diff
@@ -373,18 +373,18 @@ public struct BulkWriteResult {
      *   - insertedIds: Map of inserted IDs
      */
     fileprivate init(reply: Document, insertedIds: [Int: BSONValue?]) throws {
-        self.deletedCount = reply["nRemoved"] as? Int ?? 0
-        self.insertedCount = reply["nInserted"] as? Int ?? 0
+        self.deletedCount = try reply.getValue(forKey: "nRemoved") as? Int ?? 0
```

Comment:
Looking at this diff, I wonder if we can remove the label for `getValue()`'s first and only parameter. Perhaps this was done for consistency with `setValue()`, but asking users to repeatedly type `forKey` seems verbose. Defer to @kmahar.

I wonder if there are design patterns from other Swift libs that have getter methods (in lieu of subscript operators) where they've chosen to not used a parameter label. If so, I'd be curious to know if they did the same for their `setValue()` equivalent, too -- unless it's common Swift convention to require labels for anything with multiple required parameters.

AFAIK, the only place we mix unlabeled and labeled params is with things like `insertOne()`, where there is only one required parameter (e.g. the document) and the additional param is optional.

Reply:
I noticed this same exact thing! However, as you pointed out, this was done for consistency with `setValue()`.

I suppose we'll let @kmahar weigh in on what seems to be potentially a Swift conventions question.

Reply:
hmmm, I'm torn. I think single parameter labels *can* be useful to better explain what a method is doing. and I think "get value for key" is definitely clearer and consistent. but, I also see that it's a bit verbose. given that this method is internal, we are the only users we need to worry about, and I would choose clarity over us having to type it a few more times. 

that said, maybe it should been`for` in both these places, and not `forKey`... feels more consistent with other times prepositions are used as argument labels. encode(to:), decode(as:from:)... 

in conclusion, idk 😅

Reply:
I actually prefer `for` and I think it saves some keystrokes. Since we're uncertain, how about we just roll with `for`? Seems like a nice middle-of-the-road compromise!

Reply:
good with me

Reply:
Done (for both `setValue()` and `getValue()`)